frontend/src/views/AboutView.vue

```diff
@@ -23,19 +32,18 @@
         </p>
       </div>
 
-      <div class="mb-3">
+      <div class="useful-links-wrapper mb-5">
         <h3>Funding</h3>
         <p>
-          This project is funded by the NSF Division of Earth Sciences Award #2322510:
-          <br />
-          <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=2322510"
-            >Synthesizing hydrologic process knowledge to determine global drivers of dominant
-            processes.
+          This project is supported by the National Science Foundation (NSF), Division of Earth Sciences,
+          under Award No. 2322510:
+          <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=2322510" target="_blank">
+            Synthesizing hydrologic process knowledge to determine global drivers of dominant processes.
           </a>
         </p>
       </div>
 
-      <div class="mb-3">
+      <div class="useful-links-wrapper mb-5">
         <h3>Project Team</h3>
         <v-container fluid>
           <v-row dense>
```

Comment:
not a problem with your PR but potential for future improvement -- MD and down screen sizes have some issues:
![image](https://github.com/user-attachments/assets/d2e9d06b-9e74-4bf0-aabd-e7f970b806e7)

Reply:
also left margin:
![image](https://github.com/user-attachments/assets/f80411e8-9b65-47e7-b4ae-f1eeebdc51a9)